package-lock.json

```diff
@@ -4936,7 +4936,7 @@
       }
     },
     "packages/compute-baseline": {
-      "version": "0.2.0",
+      "version": "0.3.0",
```

Comment:
Can you update with the latest main please? This diff should not be here.

Reply:
Done, the diff no longer seems to be there.